features:
  min_gradient: 30
frame:
  base_link_id: world
  frame_id: odom
loss:
  function: None
  huber:
    c: 1.43
  tdistribution:
    v: 5.0
prediction:
  model: ConstantMotion
pyramid:
  levels: [1.0,0.5,0.25]
solver:
  max_iterations: 100
  min_step_size: 0.00000000001
keyframe_selection:
  method: idx
  idx:
    period: 5
log:
  image:
    ImageWarped:
      show: false
      block: false
    Residual:
      show: false
      block: false
    Weights:
      show: false
      block: false
    Image:
      show: false
      block: false
    Template:
      show: false
      block: false
    Depth:
      show: false
      block: false
    Tracking:
      show: false
      block: false
  plot:
    ErrorDistribution:
      show: false
      block: false
qos_overrides:
  /parameter_events:
    publisher:
      depth: 1000
      durability: volatile
      history: keep_last
      reliability: reliable
  /tf:
    publisher:
      depth: 100
      durability: volatile
      history: keep_last
      reliability: reliable
